data/templates/css/wikimedia.scss

```diff
@@ -1,5 +1,4 @@
 @import "project-setup";
-@import "compass";
```

Comment:
You'll need to go a bit farther to ditch compass from these style sheets.

`@include global-reset;` I think is compass

`@include box-sizing(border-box);` is a silly mixin for cross browser vendor prefixing we can just replace with `box-sizing: border-box;`

Then `image-height($file)` and `image-width($file)` and `image-url($file)` will all need to be replaced with actual image widths and heights for our assets and `url(../some/path/)` calls.